crates/zed/src/zed/migrate.rs

```diff
@@ -159,54 +186,51 @@ impl ToolbarItemView for MigrationBanner {
 impl Render for MigrationBanner {
     fn render(&mut self, _: &mut Window, cx: &mut Context<Self>) -> impl IntoElement {
         let migration_type = self.migration_type;
-        let file_type = match migration_type {
-            Some(MigrationType::Keymap) => "keymap",
-            Some(MigrationType::Settings) => "settings",
-            None => "",
-        };
-        let backup_file_name = self.backup_file_name();
+        let settings = ThemeSettings::get_global(cx);
+        let ui_font_family = settings.ui_font.family.clone();
 
         h_flex()
             .py_1()
             .pl_2()
             .pr_1()
-            .flex_wrap()
             .justify_between()
             .bg(cx.theme().status().info_background.opacity(0.6))
             .border_1()
             .border_color(cx.theme().colors().border_variant)
             .rounded_sm()
-            .overflow_hidden()
             .child(
                 h_flex()
                     .gap_2()
+                    .overflow_hidden()
                     .child(
                         Icon::new(IconName::Warning)
                             .size(IconSize::XSmall)
                             .color(Color::Warning),
                     )
                     .child(
-                        h_flex()
-                            .gap_0p5()
-                            .child(
-                                Label::new(format!(
-                                    "Your {} file uses deprecated settings which can be \
-                                    automatically updated. A backup will be saved to",
-                                    file_type
-                                ))
-                                .color(Color::Default),
-                            )
-                            .child(
-                                div()
-                                    .px_1()
-                                    .bg(cx.theme().colors().background)
-                                    .rounded_xs()
-                                    .child(
-                                        Label::new(backup_file_name)
-                                            .buffer_font(cx)
-                                            .size(LabelSize::Small),
-                                    ),
-                            ),
+                        div()
+                            .overflow_hidden()
+                            .text_size(TextSize::Default.rems(cx))
+                            .child(if let Some(markdown_entity) = &self.markdown_entity {
```

Comment:
We can use the method `.when_some` instead of an if-else statement. That way, you don't have to specify an else block, and it's cleaner.